charm/reactive/snap-build.py

```diff
@@ -116,6 +122,26 @@ def configure(pgsql, cache):
                 'node_env': get_node_env(environment),
                 'db_conn': pgsql.master.uri,
             })
+
+        # Render shell secrets to keep crontab cleaner.
+        render('snap-build-poller_secrets.j2', SECRETS_PATH,
```

Comment:
Should we perhaps make sure this is owned by a suitable user/group and mode 0600?

Reply:
user/group is already 'root' by default, but I am now forcing both along with perms=0600.